src/beta/resources/referral.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable no-dupe-class-members,no-unused-vars,class-methods-use-this */
+import superagent from 'superagent';
+
+import T from 'proptypes';
+import base from '../../resources/base';
+import { propTypes as userPropTypes } from '../../resources/user';
+import RequestError from '../../errors/request';
+import API from '../../easypost';
+
+export const propTypes = Object.assign({}, userPropTypes, {
+  name: T.string.isRequired, // mark as required
+  email: T.string.isRequired, // mark as required
```

Comment:
The comments stating it's required directly after the property `isRequired` doesn't help and instead muddies up the code. Let's remove these two comments.